include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -74,6 +74,8 @@ ORT_DEFINE_RELEASE(RunOptions);
 ORT_DEFINE_RELEASE(Session);
 ORT_DEFINE_RELEASE(SessionOptions);
 ORT_DEFINE_RELEASE(TensorTypeAndShapeInfo);
+ORT_DEFINE_RELEASE(SequenceTypeInfo);
```

Comment:
Thanks for the contribution - would you like to cover these with some tests ?

Reply:
Of course we would, but as far as we searched through it, there are no tests for the basic `TensorTypeAndShapeInfo` we could use to build upon.   
We could only further extend the `test_nontensor_types` to check the TypeInfos as well

Reply:
@hariharans29 The main functionality should now be covered.